mle/cli.py

```diff
@@ -59,12 +60,40 @@ def start(mode, model):
     if not check_config():
         return
 
-    if mode == 'report':
-        # Report mode
-        return report(os.getcwd(), model)
-    else:
+    if mode == 'general':
         # Baseline mode
-        return baseline(os.getcwd(), model)
+        return workflow.baseline(os.getcwd(), model)
+
+
+@cli.command()
+@click.pass_context
+@click.argument('repo', required=False)
+@click.option('--model', default=None, help='The model to use for the chat.')
+def report(ctx, repo, model):
+    """
+    report: generate report with LLM.
+    """
+    if repo is None:
+        # TODO: support local project report
+        repo = questionary.text(
+            "What is your GitHub repository? (e.g., MLSysOps/MLE-agent)"
+        ).ask()
+
+    if not re.match(r'.*/.*', repo):
+        console.log("Invalid github repository, "
+                    "Usage: 'mle report <organization/name>'")
+        return False
+
+    if not check_config():
+        # build a new project for github report generating
+        project_name = f"mle-report-{repo.replace('/', '_').lower()}"
+        ctx.invoke(new, name=project_name)
+        # enter the new project for report generation
+        work_dir = os.path.join(os.getcwd(), project_name)
```

Comment:
the project will be created at the current command execution path?

Reply:
yes, is it unreasonable for users? or we may enforce the user to new a project, before calling `mle report`.

Reply:
no, it is ok to me : ) Just want to clarify